### PR TITLE
Replace references to v2 service permissions endpoint with v3, misc c…

### PR DESCRIPTION
**Pull request by will-gant**
This documents the change introduced in cloudfoundry/cloud_controller_ng#2946, which creates a v3 endpoint equivalent to `/v2/service_instances/:guid/permissions`

Also clarifies the fact that `/v2/info` will continue to be advertised (and served) by the Cloud Controller even when the v2 API is disabled via the CAPI release.

### CODE REVIEW

**Review by stephanme — APPROVED** [review]
We should eventually get rid of any CF API v2 reference in the docs as CF API v2 is deprecated.

**Comment by will-gant** [comment]
Hi @cshollingsworth - just chasing up on this one :)

**Comment by pspinrad** [comment]
Hi @will-gant - thanks for contrib.  I'm partially covering for @cshollingsworth while she's OOO.

**Comment by pspinrad** [comment]
Oh, whoops-- I don't have write access, can't approve this PR-- will find someone who can..

**Pull request merged**
